package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "cuttle",
   "description": "The deepest card game under the sea",
-  "version": "5.1.3",
+  "version": "5.1.5",
```

Comment:
This should increment just one patch version to 5.1.4
```suggestion
  "version": "5.1.4",
```

Same applies to `package-lock.json`

It looks like the version bump was run twice

Reply:
You can either revert the last commit or just change these files by hand